bl-service/src/main/java/com/ayeeti/blservice/user/UserDTO.java

```diff
@@ -1,6 +1,22 @@
 package com.ayeeti.blservice.user;
 
 import com.ayeeti.blservice.location.LocationDTO;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor
+@Getter
+@Setter
+public class UserDTO {
```

Comment:
Why changing the source code constantly???

Reply:
Similar to the comment below, I was having hard time with the record. I needed to be able to give errorMessage dynamically.